### DISCUSSION TITLE
Alpha 181

### DISCUSSION THREAD

**Discussion by Eugeny** [Eugeny/tabby#6629] [Release Notes]
- macOS Ventura compatibility
- optimized flow control performance
- updated locales, added Indonesian and UK English
- added support for Ctrl-/ and Ctrl-@
- fixed "notify on activity" not working for the currently active tab

<hr /><em>This discussion was created from the release <a href='https://github.com/Eugeny/tabby/releases/tag/v1.0.181'>Alpha 181</a>.</em>

**Comment by maksymka** [comment]
is there instruction how install it on MacBook Pro 2017 year?

**Reply by Andy4Worc** [reply]
Or on RHEL 7? Instructions seem bleak

**Comment by mo-gd** [comment]
Very nice app !
